addons/csi-vmware-cloud-director/vcloud-csi-config.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: vcloud-csi-configmap
+  namespace: kube-system
+data:
+  vcloud-csi-config.yaml: |+
+    vcd:
+      host: {{ required "Please provide VCD_URL" .Credentials.VCD_URL }}
```

Comment:
We have a problem here. Terraform expects `VCD_URL` to have the suffix `/api`. Although the CSI driver explicitly adds this suffix "unconditionally" and that results in a malformed URL with `example.com/api/api`.